src/frontend/Komponenter/Behandling/Inngangsvilkår/Aleneomsorg/typer.ts

```diff
@@ -64,8 +64,26 @@ export interface IAnnenForelder {
     land?: string;
     dødsfall?: string;
     tidligereVedtaksperioder?: ITidligereVedtaksperioder;
+    avstandTilSøker: IAvstandTilSøker;
 }
 
+export interface IAvstandTilSøker {
+    avstandIKm: number;
```

Comment:
`avstandIKm` er nullable i backend
```suggestion
    avstandIKm?: number;
```

Reply:
Sant!